public/app/plugins/datasource/cloudwatch/plugin.json

```diff
@@ -26,6 +26,10 @@
       "name": "Grafana Labs",
       "url": "https://grafana.com"
     },
+    "keywords": [
+      "aws",
+      "amazon",
```

Comment:
oh no trailing commas I think

Reply:
oh and do we have to add the word `cloudwatch` too or do we get that one because it's in the name?

Reply:
Yeah not really needed 😄 I'll happily fix up the PR when back on Monday, but if you want in for 11.1 then please feel free to push to the branch/merge. Didn't run prettier as was just in the web UI